pallets/xcm-transactor/src/tests.rs

```diff
@@ -315,6 +315,23 @@ fn test_fee_calculation_works() {
 		})
 }
 
+// Kusama case
+#[test]
+fn test_fee_calculation_works_2() {
```

Comment:
I'd rename the test to reflect what you're actually testing.

Also, I like having a regression test with the original input as below, but a more trivial case could help too.

Reply:
A test case covering the behavior when `weight` is an even multiple of `WEIGHT_PER_SECOND` would be good, too

Reply:
we had already a test before, called `test_fee_calculation_works`. Does the fact that this test pass serve as regression test?

Reply:
I'm also tempted to add a bunch of more tests, called `test_fee_calculation_works_kusama_0_9_20_case`, `test_fee_calculation_works_polkadot_0_9_20_case`, etc..